ITEST/itest.tracing/test/org/openl/itest/RunTracingITest.java

```diff
@@ -0,0 +1,240 @@
+package org.openl.itest;
+
+import static net.mguenther.kafka.junit.EmbeddedKafkaCluster.provisionWith;
+import static net.mguenther.kafka.junit.EmbeddedKafkaConfig.brokers;
+import static org.awaitility.Awaitility.given;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.log4j.Logger;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.openl.itest.core.HttpClient;
+import org.openl.itest.core.JettyServer;
+
+import io.opentracing.Tracer;
+import io.opentracing.mock.MockSpan;
+import io.opentracing.mock.MockTracer;
+import io.opentracing.tag.Tags;
+import net.mguenther.kafka.junit.EmbeddedKafkaCluster;
+import net.mguenther.kafka.junit.EmbeddedKafkaClusterConfig;
+import net.mguenther.kafka.junit.KeyValue;
+import net.mguenther.kafka.junit.ObserveKeyValues;
+import net.mguenther.kafka.junit.SendKeyValues;
+
+public class RunTracingITest {
+
+    private static final Logger log = Logger.getLogger(RunTracingITest.class);
+    public static final int AWAIT_TIMEOUT = 1;
+
+    final String TEST_WS_URL = "/deployment1/simple1";
+    final String TEST_REST_URL = "/REST/deployment1/simple1/Hello";
+
+    private static JettyServer server;
+    private static HttpClient client;
+    private static MockTracer tracer;
+    private static EmbeddedKafkaCluster cluster;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        tracer = new MockTracer();
+        // Setting the tracer for the filter

```

Comment:
Try to register a tracer there using GlobalTracer.registerIfAbsent(tracer); instead of defining it via attributes